client/llb/fileop.go

```diff
@@ -522,23 +523,22 @@ func (a *fileActionCopy) toProtoAction(ctx context.Context, parent string, base
 }
 
 func (a *fileActionCopy) sourcePath(ctx context.Context) (string, error) {
-	p := path.Clean(a.src)
+	// filepath.Clean() also does a filepath.FromSlash(). Explicitly convert back to UNIX path
+	// separators.
+	p := filepath.ToSlash(filepath.Clean(a.src))
```

Comment:
This seems to be the last place where the behavior depends on the native OS of client/llb

Reply:
Indeed it is. This can be reverted to `path.Clean()` and we would have the same effect even on Windows.